model/src/main/kotlin/OrtResult.kt

```diff
@@ -441,6 +441,16 @@ data class OrtResult(
             emptyList()
         }
 
+    /**
+     * Retrieve all unresolved and non-excluded issues with severities equal to or over [minSeverity].
+     */
+    @JsonIgnore
+    fun getOpenIssues(minSeverity: Severity = Severity.WARNING) = collectIssues()
+        .filterNot { (id, _) -> isExcluded(id) }
+        .values
+        .flatten()
+        .filter { issue -> getResolutions().issues.none { it.matches(issue) } && issue.severity >= minSeverity }
```

Comment:
Minor optimization: The filter may be slightly more efficient if you switch the conditions, i.e. check first for the severity threshold and then for the presence of resolutions. The severity check is just a comparison while the other check is more involved.